README.md

```diff
@@ -12,10 +12,116 @@
 
 ## Installation
 
+To install the SignalFlow Python client library, open a terminal and run the
+following command:
+
 ```console
 pip install signalflow-client-python
 ```
 
+## Run a SignalFlow computation
+
+The following example allows you run a SignalFlow computation from the command
+line.
+
+1. Create a file called ``signalflow_demo.py`` that includes the following
+   content:
+
+   ```python
+   #!/usr/bin/env python
+
+   # Copyright (C) 2017 SignalFx, Inc. All rights reserved.
+   #
+   # A simple example showing how to execute a SignalFx SignalFlow computation
+   # from Python and dump its data, metadata and event output to the console.
+
+   import argparse
+   import os
+   import sys
+
+   sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", ".."))
```

Comment:
Looks like this (pre existing) code makes assumptions about its location, which would mean that this script won't work in most cases. We may need to take another look and adjust.

Reply:
I wonder whether we could overcome some of the problems that you mention here by having the user clone the GitHub repo and execute the script from the `examples` directory. It would make the page a lot cleaner – we could link to the file rather than copying the code into the README and then refer to `basic.py` in the example command.

Reply:
Yeah, up to you, but I think it would be nice to support folks who are just running their own script as well. I'll look at this script and see if I can tweak it.